### PR TITLE
telco5g: SRIOV PR baremetal job

### PR DESCRIPTION
**Pull request by sshnaidm**

### CODE REVIEW

**Comment by sshnaidm** [comment]
/pj-rehearse pull-ci-openshift-sriov-network-operator-master-e2e-telco5g-sriov

**Comment by sshnaidm** [comment]
/pj-rehearse pull-ci-openshift-sriov-network-operator-master-e2e-telco5g-sriov

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-cnftests pull-ci-openshift-sriov-network-operator-master-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse pull-ci-openshift-sriov-network-operator-master-e2e-telco5g-sriov

**Comment by sshnaidm** [comment]
/pj-rehearse pull-ci-openshift-sriov-network-operator-master-e2e-telco5g-sriov

**Comment by sshnaidm** [comment]
/pj-rehearse pull-ci-openshift-sriov-network-operator-master-e2e-telco5g-sriov

**Comment by sshnaidm** [comment]
/pj-rehearse pull-ci-openshift-sriov-network-operator-master-e2e-telco5g-sriov

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.17-e2e-telco5g-cnftests periodic-ci-openshift-release-master-nightly-4.15-e2e-telco5g

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-telco5g-sno-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-sno-cnftests

**Review by SchSeba — COMMENTED** [review]
nice work!

small comments

File: ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml [hunk 1]
```diff
@@ -200,10 +200,20 @@ tests:
         CNF_E2E_TESTS=true
         CNF_ORIGIN_TESTS=false
       T5CI_JOB_TYPE: cnftests
-      T5CI_VERSION: "4.16"
+      T5CI_VERSION: "4.17"
```
[the remark this answers is not in the document]
Comment (sshnaidm) [reply ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml:203]: Yeah, we remove them all after merge sriov jobs in all branches.

File: ci-operator/step-registry/telco5g/sriov/telco5g-sriov-workflow.metadata.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+	"path": "telco5g/sriov/telco5g-sriov-workflow.yaml",
```
[the remark this answers is not in the document]
Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/sriov/telco5g-sriov-workflow.metadata.json:2]: this is for Prow. Added.

**Review by SchSeba — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**